ansible/roles/common/templates/motd

```diff
@@ -4,7 +4,7 @@ This server brought to you by:
 \___ \| | | | '_ \___ \| '_ \ / _` |/ __/ _ \
  ___) | |_| | |_) |__) | |_) | (_| | (_|  __/
 |____/ \__,_|_.__/____/| .__/ \__,_|\___\___|
-                       |_|             v2.0.0
```

Comment:
You should consider adding some automation to this project to check for things like updating all these version number and adding things to the changelog.  I don't know of anything off the top of my head, but I've definitely seen other open source projects have bots auto-reply on PRs.  Seems like something that might be fun to do.